### PR TITLE
Multi input verbs

### PR DESCRIPTION
**Pull request by Abhinandan578**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 1]
```diff
@@ -9,17 +9,19 @@ Adobe PDF Services APIs have added support for accessing files from the external
...
 
-Accessing files from External Storage is presently restricted to the [Document Generation API](../../../apis/#tag/Document-Generation). Future releases will include the support of external storage across all PDF Services operations.
+Accessing files from External Storage is presently **not** supported in [Extract PDF](../../../apis/#tag/Extract-PDF) and [Split PDF](../../../apis/#tag/Split-PDF) API, as multi-output APIs are not allowed in External Storage Solution.
 
 ## Supported External Storage Solutions
 
 The concept of Signed URLs is common across various cloud storage solutions for downloading and uploading files. Adobe PDF Services APIs currently supports the signed URLs generated through -
 - [SharePoint](#sharepoint)
 - [Amazon S3](#amazon-s3)
+- [DropBox](#dropbox)
```
Comment (cfjedimaster) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:23]: Dropbox, not DropBox

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 2]
```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
 
 Please refer [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) for more details.
 
+## DROPBOX
```
Comment (cfjedimaster) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:39]: Dropbox

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 3]
```diff
@@ -9,17 +9,19 @@ Adobe PDF Services APIs have added support for accessing files from the external
 
 A **Signed URL** temporarily provides access to a resource. Signed URLs contain user/authentication information, allowing users without credentials to access the given resource.
 
-PDF Services APIs supports the signed URLs of input and output files in the request body. Please refer to the **External Section** in Request Body of [Document Generation API](../../../apis/#tag/Document-Generation) for more details.
+PDF Services APIs supports the signed URLs of input and output files in the request body. Please refer to the **External Section** in Request Body of each API for more details.
 
 <InlineAlert slots="text"/>
 
-Accessing files from External Storage is presently restricted to the [Document Generation API](../../../apis/#tag/Document-Generation). Future releases will include the support of external storage across all PDF Services operations.
+Accessing files from External Storage is presently **not** supported in [Extract PDF](../../../apis/#tag/Extract-PDF) and [Split PDF](../../../apis/#tag/Split-PDF) API, as multi-output APIs are not allowed in External Storage Solution.
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:16]: remove 'as multi-output APIs are not allowed in External Storage Solution.'

Comment (piyuverma) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:16]: reason is not needed to be specified

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 4]
```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
 
 Please refer [here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html) for more details.
 
+## Dropbox
+
+PDF Services APIs can connect with the Dropbox using pre-signed URLs for downloading and uploading the files to and from Dropbox location without the need for any credentials or access tokens.
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:41]: remove 'to and from'

Comment (piyuverma) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:41]: without the need for any credentials or access tokens. -> 'without the need for credentials or access token.'

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 5]
```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
...
+
+Please refer here for [downloading](https://www.dropbox.com/developers/documentation/http/documentation#files-get_temporary_link) and [uploading](https://www.dropbox.com/developers/documentation/http/documentation#files-get_temporary_upload_link) files from dropbox.
+
+## Azure Blob
+
+PDF Services APIs can connect with the Azure Blob using pre-signed URLs or [SAS Urls](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview) for downloading and uploading the files to and from Azure Blob without the need for any credentials or access tokens. Use [Best Practices](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview#best-practices-when-using-sas) when using SAS URLs.
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:47]: remove 'to and from'
without the need for any credentials or access tokens -> 'without the need for credentials or access token'

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md [hunk 6]
```diff
@@ -34,6 +36,18 @@ PDF Services APIs can connect with the Amazon S3 using pre-signed URLs for downl
...
+
+Please refer [here](https://learn.microsoft.com/en-us/azure/ai-services/translator/document-translation/how-to-guides/create-sas-tokens?tabs=Containers#create-sas-tokens-with-azure-storage-explorer) for more details. Make sure you have appropriate permissions added to your SAS tokens before generating them.
```
Comment (piyuverma) [inline src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md:49]: appropriate permissions added -> 'provided required permissions'
before generating them -> 'while generating them.'

**Review by piyuverma — CHANGES_REQUESTED** [review]

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 3 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: ok. Added auto tag also in this list

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 4 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: Done.

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 5 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: Done.

File: src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md (hunk 6 above)
Comment (Abhinandan578) [reply src/pages/overview/pdf-services-api/howtos/pdf-external-storage-sol.md]: Done.

**Pull request merged**